### DISCUSSION TITLE
Assistance Request: Academic Benefits Not Yet Received

### DISCUSSION THREAD

**Discussion by thegmitrzuk** [community/community#141002] [GitHub Education]
### Hi everyone,

I recently received an approval notification for my GitHub Education benefits on October 6, 2024. However, I haven't received access to the benefits yet, even though the approval notice mentioned they would be available soon. I understand that benefits are usually provided within 3 to 5 days after approval, but it's now past that time frame.

Could you please help me check the status of my benefits, or let me know if there are any further steps I need to take to complete this process?

**Comment by davevad93** [comment]
Hi @thegmitrzuk ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @thegmitrzuk was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by AlexKutas** [comment]
Hi guys, I have the same problem. I got Approved on October 3rd and I still don't have Academic Benefits. 
<img width="362" alt="Screenshot 2024-10-10 at 12 54 58" src="https://github.com/user-attachments/assets/e7c74a70-3c09-4616-be89-41b61d18a034">

**Reply by davevad93** [reply]
Hey @AlexKutas, given your status and the `PRO` badge in your profile, you should have access to your benefits. Try first going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and select one of the available offers, for example [Copilot](https://education.github.com/pack/redeem/copilot-student), if you can redeem it you are fine, if you can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select `I am a verified student having trouble redeeming my Global Campus offers` and explain your issue.

**Reply by AlexKutas** [reply]
Thanks. Yes, it's strange. For example:  I tried link Github Account to Termius (https://termius.com/education?utm_source=github+termius) and got next

> It looks like you haven't activated the GitHub Education program for your account.
> You may retry linking your GitHub account.

<img width="845" alt="Screenshot 2024-10-10 at 22 11 02" src="https://github.com/user-attachments/assets/96964d8d-aa42-48c7-89fb-13a2670270c1">
